src/sentry/search/events/datasets/filter_aliases.py

```diff
@@ -102,7 +102,7 @@ def project_slug_converter(
         # Create a new search filter with the correct values
         converted_filter = builder.convert_search_filter_to_condition(
             SearchFilter(
-                SearchKey("project.id"),
+                SearchKey(constants.PROJECT_ID_ALIAS),
```

Comment:
I'm not sure what other datasets may have `project_id` mapped the way you expect here. So I would recommend adding a mapping for `project.id` for metric summaries to `project_id` [here](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/snuba.py#L175) to achieve the same effect without the possibility of impacting other datasets.